.travis.yml

```diff
@@ -19,3 +19,16 @@ env:
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
 
 script : .travis.sh
+
+jobs:
+    include:
+        - stage: Upload Package
+          # We need to include the exclusion of D2 tags because this "if"
+          # replaces the global one
+          if: tag IS present
+          # Which package to deploy
+          env: DMD=dmd DIST=xenial
```

Comment:
Use a specific version in here: `DMD=2.077.*` for example, so you don't have to depend on the image's default version.